<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Б-дерево</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="searchElementSplitter">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>108</width>
      <height>43</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="searchElementField"/>
    <widget class="QPushButton" name="searchElementButton">
     <property name="text">
      <string>Найти элемент</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="eleteElementSplitter">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>108</width>
      <height>43</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="deleteElementField"/>
    <widget class="QPushButton" name="deleteElementButton">
     <property name="text">
      <string>Удалить элемент</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="addElementSplitter">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>108</width>
      <height>43</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="addElementField"/>
    <widget class="QPushButton" name="addElementButton">
     <property name="text">
      <string>Добавить элемент</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="treeorderSplitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="treeOrderLabel">
     <property name="text">
      <string>Порядок дерева</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="treeOrderSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>750</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="statusLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
